src/concourse/lib/resources.py

```diff
@@ -1,10 +1,67 @@
-from concourse.lib.models import Identifier, Resource
```

Comment:
Think what's a bit misleading/confusing about this file is why is there a `Git` model resource but no model resources for the rest of the functions?